hosts/build03/configuration.nix

```diff
@@ -17,6 +17,10 @@
     ./postgresql.nix
   ];
 
+  # agent is disabled on darwin02
+  # hercules-ci-agent: security: createProcess: posix_spawnp: does not exist
+  services.hercules-ci-agent.settings.remotePlatformsWithSameFeatures = [ "aarch64-darwin" "x86_64-darwin" ];
```

Comment:
This lets the hercules agent on build03 build jobs on darwin02 via nix distributed builds.